app/vmagent/deployment/Dockerfile

```diff
@@ -1,8 +1,8 @@
-ARG certs_image
-FROM $certs_image AS certs
-FROM scratch
-COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
-ARG src_binary
-COPY $src_binary ./vmagent-prod
+ARG base_image
+FROM $base_image
+
 EXPOSE 8429
+
 ENTRYPOINT ["/vmagent-prod"]
+ARG src_binary
+COPY $src_binary ./vmagent-prod
```

Comment:
I've reordered steps to make even empty fs-layers created by `EXPOSE/ARG/ENTRYPOINT/etc` reusable across different versions on image